README.md

```diff
@@ -6,6 +6,12 @@ Updated bundle for the browser is always attached to the GitHub Release.
 
 ![npm](https://img.shields.io/npm/v/@asyncapi/parser?style=for-the-badge) ![npm](https://img.shields.io/npm/dt/@asyncapi/parser?style=for-the-badge)
 
+
+This is a Monorepo managed using [Turborepo](https://turbo.build/) and contains the following package:
```

Comment:
I believe this should be moved up and somehow replace content from line 3 and beyond (but keeping all the details about Spectral, etc).